seaborn/categorical.py

```diff
@@ -1438,6 +1440,229 @@ def plot(self, ax):
+                **kws):
+
+        x = positions[0]
+        box_data = np.asarray(box_data)
+
+        # If we only have one data point, plot a line
+        if len(box_data) == 1:
+            ys = [box_data[0], box_data[0]]
+            xs = [x - widths / 2, x + widths / 2]
+            if vert:
+                xx, yy = ys, xs
+            else:
+                xx, yy = xs, ys
+            ax.plot(xx, yy, **kws)
+        else:
+            # Get the number of data points and calculate "depth" of
+            # letter-value plot
+            box_ends, k = self._lv_box_ends(box_data, k_depth=k_depth, p=p)
+
+            # Dictionary of functions for computing the width of the boxes
+            width_functions = {'linear' : lambda h, i, k: (i + 1.) / k,
+                               'exponential' : lambda h, i, k: 2**(-k+i-1),
+                               'area' : lambda h, i, k: (1 - 2**(-k+i-2)) / h}
+
+            # Anonymous functions for calculating the width and height
+            # of the letter value boxes
+            width = width_functions[box_widths]
+            height = lambda b: b[1] - b[0]
+
+            # Functions to construct the letter value boxes
+            def vert_perc_box(x, b, i, k, w):
+                rect = Patches.Rectangle((x - widths*w / 2, b[0]),
+                                           widths*w,
+                                           height(b), fill=True)
+                return rect
+
+            def horz_perc_box(x, b, i, k, w):
+                rect = Patches.Rectangle((b[0], x - widths*w / 2),
+                                           height(b), widths*w,
+                                           fill=True)
+                return rect
+
+            # Scale the width of the boxes so the biggest starts at 1
+            w_area = np.array([width(height(b), i, k) for i, b in enumerate(box_ends)])
+            w_area = w_area / np.max(w_area)
+
+            # Calculate the medians
+            y = np.median(box_data)
+            w = (widths + .1)
+
+            # Calculate the outliers and plot
+            outliers = self._lv_outliers(box_data, k)
+
+            if vert:
+                boxes = [vert_perc_box(x, b[0], i, k, b[1])
+                            for i, b in enumerate(zip(box_ends, w_area))]
+
+                # Plot the medians
+                ax.plot([x -  w / 2, x + w / 2], [y, y], c='k', alpha=.45, **kws)
+
+                ax.scatter(np.repeat(x, len(outliers)), outliers,
+                           marker=r"$\ast$", c=color)
+            else:
+                boxes = [horz_perc_box(x, b[0], i, k, b[1])
+                            for i, b in enumerate(zip(box_ends, w_area))]
+
+                # Plot the medians
+                ax.plot([y, y], [x -  w / 2, x + w / 2], c='k', alpha=.45, **kws)
+
+                ax.scatter(outliers, np.repeat(x, len(outliers)),
+                           marker=r"$\ast$", c=color)
```

Comment:
I didn't really have a preference as far as marker for outliers. Matplotlib uses '+' markers, so I could try to be consistent with Matplotlib or go with 'o'. Thoughts?